roles/kafka_connect/tasks/main.yml

```diff
@@ -298,6 +298,22 @@
       - kafka_connect
       - kafka_connect_parallel
       - kafka_connect_serial
+      - kafka_broker
```

Comment:
should kafka_controller groups also be added here ?

Reply:
I'll create a new PR for that with 7.4 base